service/src/main/java/com/epam/esm/service/impl/OrderServiceImpl.java

```diff
@@ -32,24 +34,14 @@ public OrderServiceImpl(OrderRepository orderRepository, UserService userService
         this.giftCertificateService = giftCertificateService;
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
     @Transactional
-    public Order create(Long userId, Long giftCertificateId) {
-        if (Objects.isNull(userId)) {
-            throw new ValidationException("User Id cannot be empty!");
-        }
-        if (Objects.isNull(giftCertificateId)) {
-            throw new ValidationException("GiftCertificate Id cannot be empty!");
-        }
+    public Order createOrder(OrderDTO dto, Authentication authentication) {
```

Comment:
You can remove user id from OrderDTO. Becuase user id can be taken from authentication